scout/server/blueprints/institutes/templates/overview/utils.html

```diff
@@ -382,20 +385,35 @@ <h3 class="header">Institute Settings</h3>
             </td>
             <td><!-- CADD score -->
               {% if variant.cadd_score %}
-                <span class="badge bg-secondary">{{ variant.cadd_score }}</span>
+                <span class="badge bg-secondary">{{ variant.cadd_score|round(2) }}</span>
```

Comment:
👍 I got a bit worried it would not exist and crash, but the construction seems common enough that this will not be the first place to give if so.